examples/shader/custom_vertex_attribute.rs

```diff
@@ -0,0 +1,150 @@
+use bevy::{
+    ecs::system::{lifetimeless::SRes, SystemParamItem},
+    pbr::MaterialPipeline,
+    prelude::*,
+    reflect::TypeUuid,
+    render::{
+        mesh::{MeshVertexAttribute, MeshVertexBufferLayout},
+        render_asset::{PrepareAssetError, RenderAsset},
+        render_resource::{
+            std140::{AsStd140, Std140},
+            BindGroup, BindGroupDescriptor, BindGroupEntry, BindGroupLayout,
+            BindGroupLayoutDescriptor, BindGroupLayoutEntry, BindingType, Buffer,
+            BufferBindingType, BufferInitDescriptor, BufferSize, BufferUsages,
+            RenderPipelineDescriptor, ShaderStages, SpecializedMeshPipelineError, VertexFormat,
+        },
+        renderer::RenderDevice,
+    },
+};
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins)
+        .add_plugin(MaterialPlugin::<CustomMaterial>::default())
+        .add_startup_system(setup)
+        .run();
+}
+
+const ATTRIBUTE_BLEND_COLOR: MeshVertexAttribute =
+    MeshVertexAttribute::new("BlendColor", 988540917, VertexFormat::Float32x4);
+
+/// set up a simple 3D scene
+fn setup(
+    mut commands: Commands,
+    mut meshes: ResMut<Assets<Mesh>>,
+    mut materials: ResMut<Assets<CustomMaterial>>,
+) {
+    // A "high" random id should be used for custom attributes to ensure proper sorting and avoid collisions with other attributes.
```

Comment:
Is this comment (and the next line) meant to be above the

```
const ATTRIBUTE_BLEND_COLOR: MeshVertexAttribute =
    MeshVertexAttribute::new("BlendColor", 988540917, VertexFormat::Float32x4);
```

line?

Reply:
Haha yup. Nice catch. I moved that value into a constant but left the comment in the original location :)